subscriber/common/job/controller.go

```diff
@@ -143,27 +143,29 @@ func NewController(params Params) *Controller {
 		params.ServiceConfig.Logger.Info("aresDB Controller is enabled")
 
 		if params.ServiceConfig.HeartbeatConfig.Enabled {
+			params.ServiceConfig.Logger.Info("heartbeat config",
+				zap.Any("heartbeat", *params.ServiceConfig.HeartbeatConfig))
 			controller.etcdServices, err = connectEtcdServices(params)
 			if err != nil {
 				panic(utils.StackError(err, "Failed to createEtcdServices"))
```

Comment:
better use logger.fatal so we can catch the log in elk

Reply:
here should stop the service to alert deployment failed via panic.

Reply:
replace with logger.panic